reqwest-retry/Cargo.toml

```diff
@@ -10,7 +10,7 @@ keywords = ["reqwest", "http", "middleware", "retry"]
 categories = ["web-programming::http-client"]
 
 [dependencies]
-reqwest-middleware = { version = "^0.1.6", path = "../reqwest-middleware" }
+reqwest-middleware = { version = "^0.1.6" }
```

Comment:
This is because we want to release it after merging this. We will set to the next development version in a following PR